warrior/Framework/Utils/encryption_utils.py

```diff
@@ -49,18 +46,22 @@ def get_key(encoded_key):
     """
     IV = None
     CIPHER = None
-    if encoded_key is False:
-        try:
+    try:
+        if encoded_key is False:
             MYFILE = Tools.__path__[0]+os.sep+"admin"+os.sep+'secret.key'
             with open(MYFILE, 'r') as myfileHandle:
                 encoded_key = myfileHandle.read()
-        except IOError:
-            print_error("Could not find the secret.key file in Tools/Admin!")
-    try:
-        IV = Random.new().read(AES.block_size)
-        CIPHER = AES.new(base64.b64decode(encoded_key), AES.MODE_CFB, IV)
-    except Exception as e:
-        print_exception("Some problem occured: {0}".format(e))
+                if not encoded_key:
```

Comment:
```if``` block can be moved out of the ```with``` block

Reply:
tried to resolve comment 'try-except-else' you have given in earlier review. moving this out, code will be similar to second review.
Here, we are trying to handle two scenarios.
1) secret.key file is missing.
2) secret.key is present, but file is empty.

please let me know if you want to move this code out. two constructs will come , 1) try-except-else for handling secret.key file missing scenario and if construct for handling empty file scenario.
do you want me to proceed with this. code will be similar to second review.